tests/suites/target_test.function

```diff
@@ -384,8 +384,7 @@ int execute_tests( int args, const char ** argv )
     while ( 1 )
     {
         ret = 0;
-        mbedtls_test_info.result = MBEDTLS_TEST_RESULT_SUCCESS;
-        mbedtls_test_info.step = (unsigned long)( -1 );
+        mbedtls_test_info_reset();
```

Comment:
Style:
```suggestion
        mbedtls_test_info_reset( );
```

Reply:
Done (with other occurrences that were found too).